### DISCUSSION TITLE
这两天发现新问题，一些视频打开后不是主页显示的up主和视频

### DISCUSSION THREAD

**Discussion by Playfulnurse** [ywmoyue/biliuwp-lite#234] [General]
![Uploading image.png…]()

**Comment by wwhdgithub** [comment]
我也遇到了，有部分首页视频跳转错误，会定向到https://b23.tv/BV1xx411c7mD。只有在首页点进去会出现这个问题，稍后再看、历史记录等打开相应链接则正常，版本[4.6.9]

**Comment by ywmoyue** [comment]
已处理了，下个版本发
